lib/improver/utilities/cube_manipulation.py

```diff
@@ -60,6 +60,14 @@ def equalise_cube_attributes(cubes, silent=None):
         UserWarning:
             If an unmatched attribute is not in the "silent" list,
             a warning will be raised.
+
+    NOTE 16/05/19: iris.experimental now has an equalise_attributes function,
```

Comment:
Does IMPRO-1143 need updating?

Reply:
IMPRO-1143 is for `compare_attributes`, which returns a list of full attribute dictionaries that don't match.  This function `equalise_cube_attributes` is different, in that it removes those mismatched attributes.  I don't think this is the only use case for `compare_attributes`, but could update the ticket to consider removing it if that's what you're thinking?